### PR TITLE
[csv-parser-price] improve parsing csv with prices

### LINKED ISSUE #809: [csv-parser-price] Remove variant-sku field from price object, add missing properties
### Description
When importing prices through impex, we serialize prices from CSV to JSON using this tool.
For example this CSV:
```csv
variant-sku,value.currencyCode,value.type,value.currencyCode,value.centAmount,value.fractionDigits,country,id
sku12,EUR,centPrecision,EUR,12900,2,DE,17405d1b-e55d-4086-a7ec-e68095702ee6
sku12,EUR,centPrecision,EUR,12900,2,LT,e43aecfd-6f94-453c-9c96-185f83224659
sku12,EUR,centPrecision,EUR,12900,2,IT,281a42d3-15cb-4169-a603-9a0fefc15ac2
```
Is being serialized to this JSON:
```json
{
   "prices" : [
      {
         "sku" : "sku12",
         "prices" : [
            {
               "value" : {
                  "centAmount" : 12900,
                  "currencyCode" : "EUR"
               },
               "id" : "17405d1b-e55d-4086-a7ec-e68095702ee6",
               "variant-sku" : "sku12",
               "country" : "DE"
            },
            {
               "value" : {
                  "currencyCode" : "EUR",
                  "centAmount" : 12900
               },
               "variant-sku" : "sku12",
               "id" : "e43aecfd-6f94-453c-9c96-185f83224659",
               "country" : "LT"
            },
            {
               "id" : "281a42d3-15cb-4169-a603-9a0fefc15ac2",
               "variant-sku" : "sku12",
               "value" : {
                  "currencyCode" : "EUR",
                  "centAmount" : 12900
               },
               "country" : "IT"
            }
         ]
      }
   ]
}
```
In this example we can see that the `csv-parser-price` module adds an extra property `variant-sku` to price object. It also does not support fields `type` and `fractionDigits`.

This later breaks price importer which compares serialized prices from CSV with existing prices on CTP API using `product-sync` from [sync-actions](https://github.com/commercetools/nodejs/tree/master/packages/sync-actions) and because the objects are different it generates update actions even when prices are equal (but enriched with an extra property)

### PR DESCRIPTION
**Pull request by junajan**
#### Summary
Fixes #809 

#### Description

This PR:
 - adds a support for optional `price.value` properties `fractionDigits` and `type`
 - removes leaked `variant-sku` from price object
 - fixes naming issue in documentation

#### Todo

- Tests
  - [x] Unit
  -~~Integration~~
  - [ ] Acceptance
- [x] Documentation

### CODE REVIEW

File: packages/csv-parser-price/package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "highland": "2.13.0",
     "lodash.mapvalues": "4.6.0",
     "lodash.memoize": "4.1.2",
+    "lodash.pick": "^4.4.0",
```
Comment (daern91) [inline packages/csv-parser-price/package.json]: Sorry for annoyance, but plz pin this one 👍

File: packages/csv-parser-price/src/main.js [hunk 2]
```diff
@@ -148,13 +155,19 @@ export default class CsvParserPrice {
...
   mergeBySku(data, currentPrice) {
     const previousPrice = data.prices[data.prices.length - 1]
-    const sku = CONSTANTS.header.sku
-    if (previousPrice && previousPrice.sku === currentPrice[sku])
-      previousPrice.prices.push(currentPrice)
+
+    // remove variant-sku property from price object
+    const {
+      [CONSTANTS.header.sku]: currentSku,
+      ..._currentPrice
```
Comment (daern91) [inline packages/csv-parser-price/src/main.js]: This naming confused me a little. Maybe `restOfCurrentPrice`?

Just something that would make it easier to understand difference between `currentPrice` and `_currentPrice`.

File: packages/csv-parser-price/src/main.js [hunk 3]
```diff
@@ -148,13 +155,19 @@ export default class CsvParserPrice {
...
+    } = currentPrice
+
+    if (previousPrice && previousPrice.sku === currentSku)
+      previousPrice.prices.push(_currentPrice)
     else
       data.prices.push({
-        sku: currentPrice[sku],
-        prices: [currentPrice],
+        sku: currentSku,
+        prices: [_currentPrice],
```
Comment (daern91) [inline packages/csv-parser-price/src/main.js]: Really enjoy the approach of this in general :)

File: packages/csv-parser-price/package.json (hunk 1 above)
Comment (junajan) [reply packages/csv-parser-price/package.json]: uhh.. forgot, sorry

**Comment by junajan** [comment]
We should support also `HighPrecisionMoney` property `preciseAmount` described [here](https://docs.commercetools.com/http-api-types.html#highprecisionmoney)

**Comment by junajan** [comment]
Here 6ba20a8 I added a support for `preciseAmount` property.

**Pull request merged**